spec/features/schools/participants/add_participants_spec.rb

```diff
@@ -15,8 +15,11 @@
   end
 
   scenario "Induction tutor can add new ECT participant" do
-    when_i_click_on_add_a_new_ect_or_mentor_link
-    then_i_am_taken_to_add_your_ect_and_mentors_page
+    when_i_am_taken_to_roles_page
```

Comment:
I find what this does surprising. Perhaps `when_i_am_on_the_roles_page`? As named, I would expect this to perform navigation

Reply:
Update will be included in subsequent PR